frontend/src/components/breadcrumb/presets.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { BreadcrumbList, BreadcrumbListItem } from '.'
+import { Text } from '../text'
+import PropTypes from 'prop-types'
+import { css } from '@emotion/core'
+
+export const BreadcrumbLi = props => (
+  <BreadcrumbListItem
+    fontSize={[2, null, 3]}
+    lineHeight={[2, null, 3]}
+    mb={[0, null, 1]}
+    fontWeight="normal"
+    {...props}
+  >
+    {props.children}
+  </BreadcrumbListItem>
+)
+
+BreadcrumbLi.propTypes = {
+  children: PropTypes.any,
+}
+
+export const Breadcrumb = props => {
+  var paddingValue
+
+  paddingValue = 0
+
+  return (
+    <BreadcrumbList
+      fontSize={[1, null, 2]}
+      lineHeight={[1, null, 2]}
+      pl={paddingValue}
+      mb={4}
+      css={css`
```

Comment:
Since list-style-type: none is set in the component itself, I think this can be removed